### PR TITLE
Initialization of project

### PR DESCRIPTION
**Pull request by marcin-jozefowicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Quick start
+
+### Backend: PostgreSQL & API
+
+You can spin up DB one easily using docker:
+
+```sh
+# use "adopttapir" as a password
+docker run --name bootzooka-postgres -p 5432:5432 -e POSTGRES_PASSWORD=adopttapir -e POSTGRES_DB=adopttapir -d postgres
```
Comment (adamw) [inline README.md:9]: do we need a database? :)

File: backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterApi.scala [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import sttp.model.StatusCode
+
+class StarterApi(http: Http, starterService: StarterService, xa: Transactor[IO]) {
+  import http._
+
+  private val starterPath = "starter.zip"
+
+  private val starterEndpoint = baseEndpoint.get
+    .in(starterPath)
+    .in(query[String](tapirVersion))
```
Comment (adamw) [inline backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterApi.scala:19]: maybe modelling this as a json body would give us more flexibility? if we introduce more parameters at some point ...

**Review by adamw — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (marcin-jozefowicz) [reply README.md]: Indeed we don't need it for now.

File: backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterApi.scala (hunk 2 above)
Comment (marcin-jozefowicz) [reply backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterApi.scala]: Sure good idea!

**Pull request merged**